src/consensus/aft/raft.h

```diff
@@ -171,13 +173,17 @@ namespace aft
       std::chrono::milliseconds view_change_timeout_,
       size_t sig_tx_interval_ = 0,
       bool public_only_ = false,
-      kv::ReplicaState initial_state_ = kv::ReplicaState::Follower,
+      std::optional<kv::LeadershipState> initial_leadership_state_ =
+        std::nullopt,
```

Comment:
Looking at `node_state.h`, this argument always seems to be set to `std::nullopt` anyway. Is it worth removing it?

Reply:
I kept it just in case, but I don't like it either. Easy enough to re-introduce if it ever becomes necessary, so I'll kill it.